corehq/apps/sso/models.py

```diff
@@ -12,11 +12,11 @@
 
 
 class IdentityProviderType:
-    AZURE_AD = 'azure_ad'
+    AZURE_AD = 'azure_ad'  # Azure AD was the original name
```

Comment:
I think you can update `IdentityProviderType.AZURE_AD` to `IdentityProviderType.ENTRA_ID`

Reply:
suggested comment:
Microsoft renamed Entra ID to Azure ID after implementing this feature

Reply:
Yes, updated in 3f1ea17